book/02-git-basics/sections/recording-changes.asc

```diff
@@ -204,7 +204,7 @@ $ cat .gitignore
 ----
 
 The first line tells Git to ignore any files ending in ``.o'' or ``.a'' – object and archive files that may be the product of building your code.
-The second line tells Git to ignore all files that end with a tilde (`~`), which is used by many text editors such as Emacs to mark temporary files.
+The second line tells Git to ignore all files that end with a tilde (`~`) in the filename, which is used by many text editors such as Emacs to mark temporary files.
```

Comment:
What if we say "all files whose names end with a tilde"? I think that's slightly smoother.